### DISCUSSION TITLE
localised leakage deactivate

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#12964] [Q&A]
I have two leakages connected through the hvac leakage option. As I try to deactivate the leakages at a certain time I get this error when running the simulation.  ERROR: VENT for ductnode has a DEVC_ID or CTRL_ID, VENT ID:Vent_01_Apartment

**Comment by drjfloyd** [comment]
10.2.3 HVAC Node Parameters

Note that a VENT being used for an HVAC system must be present throughout the simulation. The VENT
should not have a CTRL_ID or DEVC_ID. This also includes the solid surface it is attached; i.e., any OBST the
VENT is attached to also needs to be present throughout the simulation. Turning on or off a VENT connected
to an HVAC system, requires a damper in the duct connected to VENT.

**Reply by ShehabGamrah** [reply]
if I use the hvac leak option, how can I add a damper to these vents
&HVAC ID='LEAK01',TYPE_ID='LEAK',VENT_ID='Vent_01_Corridor',VENT2_ID='Vent_01_Apartment',AREA=0.0024,LEAK_ENTHALPY=.TRUE./

**Reply by drjfloyd** [reply]
Do you need to?  If your leakage is for example at the bottom of a door and the door is opened, then the pressures assocaited with each VENT for the leak path will be essentially the same and the amount of flow FDS will predict in the leak path vs. the total flow through the open door will be essentially zero.

**Comment by ShehabGamrah** [comment]
I am simulating a fire within an apartment. As the fire grows intumescent seals would expand to seal the door and prevent leakage. Hence I need to deactivate leakages after a certain time.

**Comment by drjfloyd** [comment]
The leak input is basically just a shortcut to having to define two nodes and a duct. You could instead of using the leak inputs use the normal HVAC inputs where the duct area is the leak area and you assign a duct loss of 1 in each direction.

**Comment by ShehabGamrah** [comment]
Thanks, I also have another question. For certain simulations a huge amount of smoke travels through the leakage. However for other scenarios with a similar layout and extract speed the amount of leakage through the corridor is much less.

**Reply by drjfloyd** [reply]
Leakage flows are driven by pressure differences in Pascals. Small differences in fire size, volume, etc. can easily result in 10s to 100s of Pa of difference in pressures which will result in large changes in leakage low. However, it may not be a "huge" amount of smoke. Just looking at smoke3D in Smokeview can be deceptive. Looking a slices a visibiltiy of the actual concentration of smoke is more quantitative.

**Comment by ShehabGamrah** [comment]
I really have a big problem, I can't figure out. I have one case where there is a huge amount of smoke passing through leakage. The same model settings is implemented in several other simulations before, yet this example the visibility is way lower.

**Reply by drjfloyd** [reply]
I don't know what your inputs are. I cannot help you without them.  Provide an input file that demonstrates the problem along with an explanation as to why you believe the answer is incorrect for that particular input file.

**Reply by ShehabGamrah** [reply]
Dr Floyd can I provide the input file to your email

**Reply by drjfloyd** [reply]
To the forum. I cannot agree to any non-disclosure for any information you provide me.

**Reply by mcgratta** [reply]
Plus, you should make the case so simple to demonstrate the problem that there is no chance of releasing sensitive information.

**Reply by ShehabGamrah** [reply]
Oh no thats fine, attached is the file
[lkgmodel.txt](https://github.com/firemodels/fds/files/15395618/lkgmodel.txt)

**Reply by drjfloyd** [reply]
Your case violates the restrictions for HVAC. You cannot place an HVAC VENT on a removable obstruction.

 Job TITLE        :
 Job ID string    : lkgmodel


ERROR(564): VENT: Vent_01_Corridor, is used for HVAC and attached to a removable OBST. (MPI Process:  0, CHID: lkgmodel)

ERROR: FDS was improperly set-up - FDS stopped (CHID: lkgmodel)

**Reply by ShehabGamrah** [reply]
I am using this input file to run the simulation, just checked again and I am able to run the simulation. This is a vent specified as an hvac leak connected to another vent leak. They are specified at the bottom of a door (hole).

**Reply by drjfloyd** [reply]
Then you are running an older version of FDS. This has never been allowed but we did not always trap it with an error message. Do not place an HVAC VENT on an OBST that gets removed.

**Reply by ShehabGamrah** [reply]
I am using version 6.8.0. It is attached to an hole (door) that opens then closes again. I have always used this and it works. My main problem is within this current simulation I huge quantity of smoke is pulled from the leakage yet never experienced in any other models.

**Reply by drjfloyd** [reply]
I fixed the VENT issue in your input and ran with 6.9.1.  I do not see anything wrong. A small amount of smoke leaves through the bottom of the door prior to 300 s.  At 300 s the door to the hall opens and a large amount of smoke enters the hall. At 310 s you start removing 6 m3/s in the hallway At 320 s the door closes but you are still removing 6 m3/s so the pressure in the hallway drops resulting in large flow through the leakage path than before the extraction started.

**Reply by ShehabGamrah** [reply]
My problem is beyond 400s a huge amount of smoke increases through the leak as the visibility drops. Moreover, I have always used this leakage option and visibility within the corridor is not highly influenced by the leakage as for this case a huge amount of smoke is pulled from the leakage, compared to other cases with the similar parameters

**Reply by ShehabGamrah** [reply]
Moreover, the leakage area I specified is 0.0024m2. Realistically I wouldn’t expect this amount of smoke being pulled and also the smoke from leakage increases beyond 400s. I have always used this option with previous simulations and it never showed this amount of smoke, but could it be a current bug and to resolve it, I have to move the leakage from being vertically under the door to connected horizontally between the apartment and corridor through vents placed at the floor

**Reply by drjfloyd** [reply]
You are pulling 6 m3/s out of the hallway.  Results from FDS 6.9.1 show the pressure drop across the apartment door is 25 Pa once the hallway extraction begins and the apartment door is closed. That is not an unreasonable amount of pressure. 25 Pa is a leak velocity of ~8 m/s at the temperatures in the fire room near the leak. For an 0.0024 m2 leak FDS shoud give ~0.017 m3/s of flow which is the flow FDS is predicting.  That is not a large volume flow of smoke.

**Reply by ShehabGamrah** [reply]
I am assessing the conditions based on visibility. I don't understand how come the leakage causes the visibility to drop below 10. As I have run this simulation without leakage and visibility increased to 30. Also, I have always used the leakage option with 6m3/s and this exaggerated amount of smoke was never pulled from the leakage

**Reply by drjfloyd** [reply]
You have access to all the quantities being predicted in FDS via outputs. If this case is significantly different than you other cases, figure out why. Look at the leakage vent pressures and flows, the pressures in the domain, etc.  FIgure out what the difference is, and then determine if it is likely an error in FDS or just an error in your assumptions of what the results should be. As it stands now, a 25 Pa pressure drop does not seem excessive given the exhaust flow rate, the area available for makeup flow, and the flow path that make up takes.

**Comment by ShehabGamrah** [comment]
I have several other cases with a higher pressure drop across the door and not this amount of smoke being pulled from the leakage. Also suddenly after visibility is getting improved then a purge of smoke leaks from the apartment. I hope I can do a quick call to clarify further and show previous cases.

**Reply by drjfloyd** [reply]
Examine your outputs. Figure out what is changing. Then post relevant cases with a detailed explanation of what you think is wrong and why with a quantative explanation here.

**Reply by ShehabGamrah** [reply]
The images demonstrate the comparison between the case I sent and a similar case with the same exhaust speed and sources of replacement air. As shown below the amount of smoke in the corridor is reasonable from the apartment to the extract position compared to the other one. That's why I need to understand why this is happening for this specific case. Also pressure difference across door was higher for the better visibility case. 


![image](https://github.com/firemodels/fds/assets/96638521/18fccaa6-27ce-4e68-853e-3ad0c1a917f2)
![image](https://github.com/firemodels/fds/assets/96638521/b63c8cb8-72a7-4260-8d1a-084cf0661011)
![image](https://github.com/firemodels/fds/assets/96638521/f3286025-3bc1-4239-a1ed-7a5193710fbc)
![image](https://github.com/firemodels/fds/assets/96638521/1b213bd7-fae6-4137-a719-e660d27ebc7b)

**Reply by drjfloyd** [reply]
The case you provide is not showing abberant behavior.  These two sets of images are not the same geometry; you should not expect identical results. Again you have access to all the quantities being predicted by FDS, and you are just focusing here on visbility. Why is there a visiblity difference? What is it about the differences in these goemetries that is causing the different results? You are the modeler; do the work to figure out what is resulting in this difference.
